reframe/core/meta.py

```diff
@@ -417,19 +433,8 @@ class attribute. This behavior does not apply when the assigned value
         is not allowed. This would break the parameter space internals.
         '''
 
-        # Set the value of a variable (except when the value is a descriptor).
         try:
-            var_space = super().__getattribute__('_rfm_var_space')
-            if name in var_space:
-                if not hasattr(value, '__get__'):
-                    var_space[name].define(value)
-                    return
-                elif not var_space[name].field is value:
-                    desc = '.'.join([cls.__qualname__, name])
-                    raise ReframeSyntaxError(
-                        f'cannot override variable descriptor {desc!r}'
-                    )
-
+            cls.setvar(name, value)
         except AttributeError:
             pass
```

Comment:
Perhaps it's now worth replacing this `pass` by a comment here saying that `setvar` will raise an `AttributeError` when unsuccessful at setting the variable's value.

Reply:
I've replaced also the `pass` later on with a comment and added a couple more comments.